client/kie-client/src/main/java/org/apache/servicecomb/kie/client/http/HttpTransportImpl.java

```diff
@@ -0,0 +1,116 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.servicecomb.kie.client.http;
+
+import org.apache.http.client.HttpClient;
+import org.apache.http.client.methods.*;
+import org.apache.http.entity.StringEntity;
+import org.apache.http.impl.client.HttpClients;
+import org.apache.http.util.EntityUtils;
+
+import java.io.IOException;
+
+/**
+ * Created by   on 2019/10/16.
+ */
+public class HttpTransportImpl implements HttpTransport {
+
+    private static final String HEADER_CONTENT_TYPE = "Content-Type";
+
+    private static final String HEADER_USER_AGENT = "User-Agent";
+
+    private static final String HEADER_TENANT_NAME = "x-domain-name";
+
+    protected HttpClient httpClient;
+
+    public HttpClient getHttpClient() {
+        return httpClient;
+    }
+
+    public void setHttpClient(HttpClient httpClient) {
+        this.httpClient = httpClient;
+    }
+
+    public HttpTransportImpl() {
+        httpClient = HttpClients.createDefault();
```

Comment:
Need to create a task to implement TLS. A lot of security settings should be set.